src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -1496,21 +1504,24 @@ private static BoundExpression GetDefaultParameterSpecialNoConversion(SyntaxNode
             else if (parameter.IsIUnknownConstant)
             {
                 // new UnknownWrapper(default(object))
-                var methodSymbol = (MethodSymbol)compilation.GetWellKnownTypeMember(WellKnownMember.System_Runtime_InteropServices_UnknownWrapper__ctor);
+                var methodSymbol = (MethodSymbol?)compilation.GetWellKnownTypeMember(WellKnownMember.System_Runtime_InteropServices_UnknownWrapper__ctor);
+                Debug.Assert(methodSymbol is { });
```

Comment:
>Debug.Assert(methodSymbol is { }); [](start = 16, length = 34)

Could the well-known type member be missing?

Reply:
I'll file an issue for a possible bug here.

---
In reply to: [386623400](https://github.com/dotnet/roslyn/pull/42068#discussion_r386623400) [](ancestors = 386623400)